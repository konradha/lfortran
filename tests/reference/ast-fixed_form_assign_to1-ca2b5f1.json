{
    "basename": "ast-fixed_form_assign_to1-ca2b5f1",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_assign_to1.f",
    "infile_hash": "d76077afdd3f3c22b5e6769f2365e819c0fe06298487bd65c6e59526",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixed_form_assign_to1-ca2b5f1.stdout",
    "stdout_hash": "a9cb71ad9d399d3cf7eeeefd6027da95d2a153a24d93f317d112886e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}