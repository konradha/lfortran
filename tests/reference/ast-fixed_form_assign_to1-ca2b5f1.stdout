(TranslationUnit 
    [(Program 
        assign3 
        () 
        [] 
        [] 
        [] 
        [(Assign 
            10 
            10 
            next 
            ()
        )] 
        []
    )]
)
